TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/workflow/commands/status.py

```diff
@@ -0,0 +1,376 @@
+"""Status command for workflow CLI - shows detailed status of workflows."""
+
+import logging
+import os
+import click
+from rich.console import Console
+from rich.tree import Tree
+
+from ..models.utils import ExitCode
+from ..services.workflow_service import WorkflowService
+
+logger = logging.getLogger(__name__)
+
+
+@click.command(name="status")
+@click.argument('workflow_name', required=False)
+@click.option(
+    '--argo-server',
+    default=f"http://{os.environ.get('ARGO_SERVER_SERVICE_HOST', 'localhost')}"
+    f":{os.environ.get('ARGO_SERVER_SERVICE_PORT', '2746')}",
+    help='Argo Server URL (default: auto-detected from Kubernetes service env vars, or ARGO_SERVER env var)'
+)
+@click.option(
+    '--namespace',
+    default='ma',
+    help='Kubernetes namespace for the workflow (default: ma)'
+)
+@click.option(
+    '--insecure',
+    is_flag=True,
+    default=False,
+    help='Skip TLS certificate verification'
+)
+@click.option(
+    '--token',
+    help='Bearer token for authentication'
+)
+@click.option(
+    '--all',
```

Comment:
Is there a command to show me the list of all the workflows, present and historical?  Can we easily make tab-complete expand with the list of workflows - maybe with a --previous NAME option?